PROVIDERS.rst

```diff
@@ -251,3 +272,48 @@ The suspension may be triggered by any committer after the following criteria ar
 
 The suspension will be lifted when the dependencies of the provider are made compatible with the Apache
 Airflow and with other providers - by merging a PR that removes the suspension and succeeds.
+
+Removing community providers
+----------------------------
+
+The providers can be removed from main branch of Airflow when the community agrees that there should be no
+more updates to the providers done by the community - except maybe potentially security fixes found. There
+might be various reasons for the providers to be removed:
+
+* the service they connect to is no longer available
+* the dependencies for the provider are not maintained any more and there is no viable alternative
+* there is another, more popular provider that supersedes community provider
+* etc. etc.
+
+Each case of removing provider should be discussed individually and separate ``[VOTE]`` thread should start,
+where regular rules for code modification apply (following the
+`Apache Software Foundation voting rules <https://www.apache.org/foundation/voting.html#votes-on-code-modification>`_).
+In cases where the reasons for removal are ``obvious``, and discussed before, also ``[LAZY CONSENSUS]`` thread
+can be started. Generally speaking a discussion thread ``[DISCUSS]`` is advised before such removal and
+sufficient time should pass (at least a week) to give a chance for community members to express their
+opinion on the removal.
+
+There are the following consequences (or lack of them) of removing the provider:
+
+* One last release of the provider is done with documentation updated informing that the provider is no
+  longer maintained by the Apache Airflow community - linking to this page. This information should also
+  find its way to the package documentation and consequently - to the description of the package in PyPI.
+* An ``[ANNOUNCE]`` thread is sent to the devlist and user list announcing removal of the provider
+* The released provider packages remain available on PyPI and in the Downloads/Archives of the Apache
+  Software Foundation, also it remains in the Index of the Apache Airflow Providers at
```

Comment:
Do we keep it in svn?

Reply:
Good point.

Indeed SVN finally gets to "Downloads/Archive": https://downloads.apache.org/airflow/ you can see latest release there and https://archive.apache.org/dist/airflow/  include all historical releases. 

We cannot remove things from archives (and that's fine), but indeed we can (and probably should) releave it from SVN which will remove it from "downloads".

Reply:
Updated/Clarified.